docs/releases/v2018.2.1.rst

```diff
@@ -34,6 +34,10 @@ ramips-mt76x8
   - Archer C50 (v3, v4) [#noibss]_
   - TL-WR841N (v13) [#noibss]_
 
+.. [#noibss]
+  Device or target does not support AP+IBSS mode: This device or target will not be built
+  when *GLUON_WLAN_MESH* is set to ``ibss``.
+
```

Comment:
i think this is out of scope of this PR and should be done separately/beforehand for all affected release notes

Reply:
It is in scope so far as this generates an error when generating the gettext po files. So while not directly in scope it fixes errors that are made visible by this change. But I'm happy to pull this out of the PR.